charts/seed-controlplane/charts/etcd/templates/etcd-hvpa.yaml

```diff
@@ -69,9 +69,16 @@ spec:
               maxAllowed:
                 memory: 30G
                 cpu: "4"
+{{- if eq .Values.role "main"}}
```

Comment:
Can you do this in the code instead because there we already have the distinction of main and events.
https://github.com/gardener/gardener/blob/master/pkg/operation/botanist/controlplane.go#L1067

Reply:
Added in the code. PTAL @timuthy